doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -659,16 +659,18 @@ Modem libraries
 
 * :ref:`nrf_modem_lib_readme`:
 
+  * Updated the RTT trace backend to allocate the RTT channel at boot, instead of when the modem is activated.
   * Rename the nRF91 socket offload layer from ``nrf91_sockets`` to ``nrf9x_sockets`` to reflect that the offload layer is not exclusive to the nRF91 Series SiPs.
+  * Removed support for deprecated RAI socket options ``SO_RAI_LAST``, ``SO_RAI_NO_DATA``, ``SO_RAI_ONE_RESP``, ``SO_RAI_ONGOING``, and ``SO_RAI_WAIT_MORE``.
 
 * :ref:`modem_info_readme` library:
 
   * Fixed a potential issue with scanf in the :c:func:`modem_info_get_current_band` function, which could lead to memory corruption.
 
-* :ref:`nrf_modem_lib_readme` library:
+* :ref:`pdn_readme` library:
 
-  * Updated the RTT trace backend to allocate the RTT channel at boot, instead of when the modem is activated.
-  * Removed support for deprecated RAI socket options ``SO_RAI_LAST``, ``SO_RAI_NO_DATA``, ``SO_RAI_ONE_RESP``, ``SO_RAI_ONGOING``, and ``SO_RAI_WAIT_MORE``.
+  * Added the event ``PDN_EVENT_CTX_DESTROYED`` to indicate when a PDN connection for a PDP context is destroyed.
```

Comment:
If I understand this correctly, the purpose of the new event is to indicate, that the PDP context (i.e. the configuration) has been destroyed/removed. Naturally, all PDN connections are always disconnected when modem is switched to flight mode (`+CFUN=4`) or minimal functionality mode (`+CFUN=0`). The text should only talk about the context, not the connection, I think.

Reply:
Thanks, I'll update the release note.